src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -2129,4 +2129,13 @@ Widening and user defined conversions are not supported with generic types.</val
   <data name="UseStringMethodCharOverloadWithSingleCharactersTitle" xml:space="preserve">
     <value>Use char overload</value>
   </data>
+  <data name="AvoidSingleUseOfLocalJsonSerializerOptionsMessage" xml:space="preserve">
+    <value>Avoid single-use of local 'JsonSerializerOptions' as it is suboptimal, use a singleton instance instead to improve performance</value>
+  </data>
+  <data name="AvoidSingleUseOfLocalJsonSerializerOptionsDescription" xml:space="preserve">
+    <value>Avoid single-use of local 'JsonSerializerOptions' as it is suboptimal, use a singleton instance instead to improve performance.</value>
+  </data>
+  <data name="AvoidSingleUseOfLocalJsonSerializerOptionsTitle" xml:space="preserve">
+    <value>Avoid single-use of local 'JsonSerializerOptions' as it is suboptimal</value>
```

Comment:
Something like this sounds better to me, CC @gewarren 
```suggestion
    <value>Prefer reusing 'JsonSerializerOptions' instances</value>
```

Reply:
```suggestion
    <value>Reuse 'JsonSerializerOptions' instances</value>
```